src/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -89,7 +89,9 @@ public static void Run(Context context)
 
             var compositeFilter = new CompositeFilter()
                 .Add<ImplicitSymbolsFilter>()
-                .Add(new SymbolAccessibilityBasedFilter(context.IncludeVisibleOutsideOfAssembly));
+                .Add(new SymbolAccessibilityBasedFilter(
+                    context.IncludeVisibleOutsideOfAssembly,
+                    /*IncludeEffectivelyPrivateSymbols*/true));
```

Comment:
There's an in-built syntax for that in C# that avoids comment as they can get out-of-sync soon (paramName: value):

L94:
```
includeEffectivelyPrivateSymbols: true));
```